types/packageMeta.ts

```diff
@@ -5,7 +5,12 @@ export interface PackageMetaInterface {
       fileCount: number;
       unpackedSize: number;
     };
-    license: string;
+    license?: Partial<LicenseInterface> | string | null;
```

Comment:
Hmm ... do we really need to have "null" here? I think if the license is undefined it is enough 🙃

Reply:
I agree and I thought of that at first, however there are subtle differences between undefined and null :)
Also, the reason here is because *formatLicense()* may actually return null, which is better than empty string
`https://github.com/verdaccio/ui/blob/4.x-master/src/utils/package.ts#L20-L30`

Reply:
Hi! Yes there are differences between null and undefinied :-) in this case I would update _formatLicense()_ to return undefinied...you can write as below:

`export function formatLicense(license: any): string | undefined {
  if (isString(license)) {
    return license;
  }

  if (license && isObject(license) && license.type) {
    return license.type;
  }

  return ;
}`

I have also seen that the license is been used here - only:
![image](https://user-images.githubusercontent.com/29228205/60766823-67a37f80-a0af-11e9-937c-ec1ad30b1397.png)

So undefined will be better